packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -944,10 +950,10 @@ describe('<Slider />', () => {
     });
 
     expect(handleChange.callCount).to.equal(1);
-    expect(handleChange.firstCall.returnValue).to.deep.equal({
-      name: 'change-testing',
-      value: 4,
-    });
+    const returnValue = handleChange.firstCall.returnValue;
+    expect(returnValue.name).to.equal('change-testing');
+    expect(returnValue.value).to.equal(4);
+    expect(returnValue.event).to.not.equal(undefined);
```

Comment:
That's not very helpful. I positive assertion would help more. Also
```suggestion
    expect(returnValue.event).not.to.equal(undefined);
```

Reply:
Oh wow, I have never noticed the difference. In the codebase, we have

- `).to.not.` x149
- `).not.to.` x322

Happy to consolidate on the latter.